### DISCUSSION TITLE
KaTeX block notation does not take effect

### DISCUSSION THREAD

**Discussion by sharetown** [nunocoracao/blowfish#156] [Q&A] [answered]
I tried to introduce the KaTeX block notation of the example into the newly created markdown document, but it did not take effect after copying the example content directly. Do I need to enable what parameter configuration?

**Comment by nunocoracao** [comment, marked as answer]
Hello @sharetown,

from the current documentation:

"Blowfish will only bundle the KaTeX assets into your project if you make use of mathematical notation. In order for this to work, simply include the [katex shortcode](https://nunocoracao.github.io/blowfish/docs/shortcodes/#katex) within the article. Any KaTeX syntax on that page will then be automatically rendered." 

You need to include the {{< katex >}} shortcode at the beginning of each file you want to use katex in. 

Already added that to the docs with an example to make sure future users don't miss that. Thanks for pointing it out.

**Reply by sharetown** [reply]
That's right, thank you very much
